### PR TITLE
RIP-69, better living through linting

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira-secure.berkeley.edu/browse/RIP-69

Lint it good.

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -52,22 +46,22 @@ const routes:RouteRecordRaw[] = [
...
-        component: NotFound,
+        component: () => import('@/views/NotFound.vue'),
         meta: {
           title: 'Page not found'
         }
       },
       {
         path: '/error',
-        component: Error,
+        component: () => import('@/views/Error.vue'),
```
Comment (johncrossman) [inline src/router.ts:64]: I brought back lazy-loading in `router.ts`; recent PRs did not fix mime-type problem. After this is merged I will flip dev CodePipeline to use my own working branch to squash that bug. 🪰

**Comment by pauline2k** [comment]
Linters are happy but build-vue is not.

**Comment by lyttam** [comment]
Travis says: `[vite:load-fallback] Could not load /home/travis/build/ets-berkeley-edu/ripley/src/Login.vue (imported by src/router.ts): ENOENT: no such file or directory, open '/home/travis/build/ets-berkeley-edu/ripley/src/Login.vue'`

**Comment by johncrossman** [comment]
Thank you @pauline2k and @lyttam .  PR amended

**Pull request merged**